web/vibe/web/web.d

```diff
@@ -183,7 +183,16 @@ URLRouter registerWebInterface(C : Object, MethodStyle method_style = MethodStyl
 		/*static if (isInstanceOf!(SessionVar, __traits(getMember, instance, M))) {
 			__traits(getMember, instance, M).m_getContext = toDelegate({ return s_requestContext; });
 		}*/
-		static if (!is(typeof(__traits(getMember, Object, M)))) { // exclude Object's default methods and field
+
+		// Ignore special members, such as ctor, dtors, postblit, and opAssign
+		// See https://github.com/vibe-d/vibe.d/issues/2438
+		static if (M == "__ctor" || M == "__dtor" || M == "__xdtor" ||
+				   M == "__postblit" || M == "__xdtor" || M == "__xpostblit" ||
+				   M == "opAssign")
+		{ /* Ignore */ }
+
+		// exclude Object's default methods and field
+		else static if (!is(typeof(__traits(getMember, Object, M)))) {
```

Comment:
This seems backwards. What about combining both static ifs?

```suggestion

		// Ignore special members, such as ctor, dtors, postblit, and opAssign
		// See https://github.com/vibe-d/vibe.d/issues/2438
		// exclude Object's default methods and field
		static if (M != "__ctor" && M != "__dtor" && M != "__xdtor" &&
				   M != "__postblit" && M != "__xdtor" && M != "__xpostblit" &&
				   M != "opAssign" &&
				   !is(typeof(__traits(getMember, Object, M)))) {
```

Reply:
IMO it only seems backward if you don't see it as a `continue` ;)
Personally I prefer to "get things out of the way", using `continue`, early return, etc, and this is why I originally wrote it this way. But I don't mind either way, so suggestion applied.